src/messages/sms/schedule.ts

```
@@ -0,0 +1,42 @@
+import { LayloSegmentConfiguration } from "fans/segments/getSegment";
+import { sendMessageToApi } from "./lib/sendMessageToApi";
+import { configuration } from "config";
```

ESLint

[error] 'configuration' is defined but never used.

(@typescript-eslint/no-unused-vars)